python/lsst/cp/pipe/ptc.py

```diff
@@ -404,10 +404,15 @@ def runDataRef(self, dataRef, visitPairs):
         allTags = []
         for (v1, v2) in visitPairs:
             # Get postISR exposures.
-            dataRef.dataId['expId'] = v1
-            exp1 = dataRef.get("postISRCCD", immediate=True)
-            dataRef.dataId['expId'] = v2
-            exp2 = dataRef.get("postISRCCD", immediate=True)
+            try:
+                dataRef.dataId['expId'] = v1
+                exp1 = dataRef.get("postISRCCD", immediate=True)
+                dataRef.dataId['expId'] = v2
+                exp2 = dataRef.get("postISRCCD", immediate=True)
+            except RuntimeError:
+                self.log.warn(f"postISR exposure for either visit {v1} or visit {v2} could not be retreived. "
```

Comment:
This should probably not use "visit" anymore, as these should all be expId values now.  There are a lot of them already in place, but we should at least not add any more.